packages/loader/container-loader/src/container.ts

```diff
@@ -1102,7 +1098,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         // Save attributes for the document
         const documentAttributes = {
-            branch: this.id,
+            // branch: this.id,
```

Comment:
please remove it, there seems to be no reason to leave it commented

Reply:
also need to update snapshot test for this

Reply:
i will create a separate pr for this

Reply:
#8840